src/Sylius/Bundle/ResourceBundle/EventListener/ResourceDeleteSubscriber.php

```diff
@@ -161,4 +155,14 @@ private function isHtmlRequest(Request $request)
     {
         return 'html' === $request->getRequestFormat();
     }
+
+    /**
+     * @param $resource
+     *
+     * @return string
+     */
+    protected function castResourceToString($resource)
+    {
+        return (string) new StringConverter($resource);
```

Comment:
Is it required to cast result to ``string``? I think that ``StringConverter::_toString()`` **must** return ``string`` value;